grpc/src/ugrpc/impl/protobuf_utils.hpp

```diff
@@ -3,13 +3,24 @@
 #include <cstddef>
 #include <string>
 
-#include <userver/field_options.pb.h>
+namespace google::protobuf {
+
+class Message;
+class FieldDescriptor;
+
+}  // namespace google::protobuf
+
+namespace userver {
+
+class FieldOptions;
+
+}  // namespace userver
```

Comment:
that is not correct in some cases, because USERVER_NAMESPACE does not always equal to `userver`

you should declare the class below the `USERVER_NAMESPACE_BEGIN`

Reply:
But this is not USERVER_NAMESPACE, is it? This namespace corresponds to the name of the package in .proto, which is always userver: https://github.com/userver-framework/userver/blob/develop/grpc/proto/userver/field_options.proto

Reply:
sorry, okey))